src/main.ts

```diff
@@ -211,7 +212,10 @@ const command: Command = {
 				files: [
 					'./webpack.config.js'
 				]
-			}
+			},
+			hints: [
+				'to build run ' + underline('webpack -config ./config/build-webpack/webpack.config.js')
```

Comment:
fair point, isn't that how most people have it set up? I could change it to ref the one in `node_modules` if you prefer?